src/java.desktop/windows/native/libawt/windows/awt_List.cpp

```diff
@@ -171,16 +171,16 @@ void AwtList::Reshape(int x, int y, int w, int h)
 {
     AwtComponent::Reshape(x, y, w, h);
 
-/*
-    HWND hList = GetListHandle();
-    if (hList != NULL) {
-        long flags = SWP_NOACTIVATE | SWP_NOZORDER | SWP_NOCOPYBITS;
-        /*
-         * Fix for bug 4046446.
-         * /
-        SetWindowPos(hList, 0, 0, 0, w, h, flags);
-    }
-*/
+//
+//  HWND hList = GetListHandle();
+//  if (hList != NULL) {
+//      long flags = SWP_NOACTIVATE | SWP_NOZORDER | SWP_NOCOPYBITS;
+//      /*
+//       * Fix for bug 4046446.
+//       */
+//      SetWindowPos(hList, 0, 0, 0, w, h, flags);
+//  }
+//
```

Comment:
Maybe it's time to remove this commented out code? It's been commented out since 2007 when the code moved to Mercurial.

If you decide to remove it, please submit a new issue to communicate the intention clearly.

Reply:
Perhaps, but I'll leave that to the discretion of the client libs group. Are the new changes fine with you though?

Reply:
I'd rather remove the offending commented out code than change its formatting.

Could you submit a bug, please, and assign it to me?

There's a closed test associated with [JDK-4046446](https://bugs.openjdk.org/browse/JDK-4046446), it can be open-sourced. Then it'll be safe to remove the commented out code.

Reply:
Alright here you go :) https://bugs.openjdk.org/browse/JDK-8301994

Reply:
> Alright here you go :) https://bugs.openjdk.org/browse/JDK-8301994

Thank you, @TheShermanTanker.

Do you still want to keep the changes?
If it's not too critical, I would rather prefer you revert the changes to `awt_List.cpp`.

Reply:
No worries, I'm fine with reverting `awt_List.cpp`, I can set my local branch up to avoid a rebase conflict when the commented code does get deleted rather easily

On the other hand, could I trouble you for a quick review of https://github.com/openjdk/jdk/pull/12405? It's just a constructor from libawt that I happened to overlook in my previous Pull Request on initialization orders. Sorry for the trouble!

Reply:
Thanks for the review! Are the other changes here ok with you in the meantime?